Products/ZenUI3/browser/resources/js/zenoss/form/TimeSpan.js

```diff
@@ -0,0 +1,54 @@
+/*****************************************************************************
+ *
+ * Copyright (C) Zenoss, Inc. 2015, all rights reserved.
+ *
+ * This content is made available according to terms specified in
+ * License.zenoss under the directory where your Zenoss product is installed.
+ *
+ ****************************************************************************/
+
+
+/* package level */
+(function() {
+    Ext.define("Zenoss.form.TimeSpan", {
```

Comment:
This widget has a number and a timespan (weeks months and days). It is used for inputting the amount of data to fetch as well as the amount of time to project to see if the threshold is breached.